doc/source/man.rst

```diff
@@ -32,206 +32,217 @@ rho performs four major tasks:
+A network profile defines a collection of network information, including IP addresses or host names, SSH ports, and SSH credentials. The SSH credentials are provided through reference to one or more authentication profiles. A discovery scan can reference a network profile so that the act of running the scan is automatic and repeatable, without a requirement to reenter network information for each scan attempt.
 
-Creating and Editing Profiles
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Creating and Editing Network Profiles
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-A profile is essentially a concise collection of the information that rho needs to connect to a network or system. This means it contains servers to connect to and authentication credentials to use. Each of these parameters allowed multiple entries, so the same profile can access a patchwork of servers and subnets, as needed.
+To create a network profile, supply one or more hosts or IP addresses to connect to with the ``--hosts`` option and the authentication profiles needed to access those systems with the ``--auth`` option. The ``rho profile`` command allows multiple entries for each of these options. Therefore, a single network profile can access a collection of servers and subnets as needed to create an accurate and complete scan.
 
 **rho profile add --name=** *name* **--hosts** *ip_address* **--auth** *auth_profile* **[--sshport=** *ssh_port* **] [--vault=** *vault_file* **]**
 
 ``--name=name``
 
-  This required argument sets the name of the new profile. This name is used to identify the profile in later operations. Use a descriptive name, such as ``"ColoSubnet"``.
+  Required. Sets the name of the new network profile. Use a descriptive name that is meaningful to your organization, such as ``APSubnet`` or ``Lab3``.
 
 ``--hosts ip_address``
 
-  This sets the IP address, hostname, or IP address range to use when running discovery. You may provide a list of hosts or a file where each item is on a separate line. There are several different formats that are allowed for the *ip_address* value.
+  Sets the host name, IP address, or IP address range to use when running a scan. You can also provide a path and file name for a file that contains a list of hosts or IP addresses or ranges, where each item is on a separate line. The following examples show several different formats that are allowed as values for the ``--hosts`` option:
 
-  1. A specific hostname:
+  * A specific host name:
 
     --hosts server.example.com
 
-  2. A specific IP address:
+  * A specific IP address:
 
-    --hosts 1.2.3.4
+    --hosts 192.0.2.19
 
-  3. An IP address range:
+  * An IP address range:
 
-    --hosts "1.2.3.[4:255]"
+    --hosts 192.0.2.[1:254]
+
+  * A file:
+
+    --hosts /home/user1/hosts_file
```

Comment:
Should we mention about the CIDR notation? For example `192.0.2.0/24` will produce the same as the range `192.0.2.[0:255]`

Reply:
This might be a duplicate like the following comment. 

Change made:
* An IP address range:

    --hosts 192.0.2.[0:255]
    or
    --hosts 192.0.2.0/24